views/networking/load_balancer/create.erb

```diff
@@ -0,0 +1,177 @@
+                  locals: {
+                    name: "algorithm",
+                    label: "Load Balancing Algorithm",
+                    options: ["round-robin", "hash-based"].map { |a| [a.tr("-", "_"), a] },
+                  }
+                ) %>
+              </div>
+              <div class="sm:col-span-1">
+                <%== render(
+                  "components/form/text",
+                  locals: {
+                    name: "health_check_endpoint",
+                    label: "Health Check Endpoint",
+                    attributes: {
+                      required: true,
+                      placeholder: "/up"
+                    }
+                  }
+                ) %>
+              </div>
+            </div>
+            <div class="mt-6 grid grid-cols-1 gap-x-6 gap-y-8 sm:grid-cols-6">
+              <div class="sm:col-span-1">
+                <%== render(
+                  "components/form/text",
+                  locals: {
+                    name: "health_check_interval",
+                    label: "Health Check Interval",
+                    attributes: {
+                      required: true,
+                      placeholder: "30"
+                    }
+                  }
+                ) %>
+              </div>
+              <div class="sm:col-span-1">
+                <%== render(
+                  "components/form/text",
+                  locals: {
+                    name: "health_check_timeout",
+                    label: "Health Check Timeout",
+                    attributes: {
+                      required: true,
+                      placeholder: "10"
+                    }
+                  }
+                ) %>
+              </div>
+            </div>
+            <div class="mt-6 grid grid-cols-1 gap-x-6 gap-y-8 sm:grid-cols-6">
+              <div class="sm:col-span-1">
+                <%== render(
+                  "components/form/text",
+                  locals: {
+                    name: "health_check_up_threshold",
+                    label: "Health Check Healthy Threshold",
+                    attributes: {
+                      required: true,
+                      placeholder: "3"
+                    }
+                  }
+                ) %>
+              </div>
+              <div class="sm:col-span-1">
+                <%== render(
+                  "components/form/text",
+                  locals: {
+                    name: "health_check_down_threshold",
+                    label: "Health Check Unhealthy Threshold",
+                    attributes: {
+                      required: true,
+                      placeholder: "3"
+                    }
+                  }
+                ) %>
+              </div>
+            </div>
+            <div class="mt-6 grid grid-cols-1 gap-x-6 gap-y-8 sm:grid-cols-6">
+              <div class="sm:col-span-2">
+                <%== render(
+                  "components/form/select",
```

Comment:
Since most other inputs have default values, and this one is important, should we move it to the top, just below the name input?